README.md

```diff
@@ -63,3 +70,28 @@ module "app_ecs_cluster" {
 | ecs\_instance\_role | The name of the ECS instance role. |
 
 <!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
+
+## Developer Setup
+
+Install dependencies (macOS)
+
+```shell
+brew install pre-commit go terraform terraform-docs
```

Comment:
yet another thing missing from the template 
```suggestion
brew install pre-commit go terraform terraform-docs
pre-commit install --install-hooks
```

Reply:
PR in the template https://github.com/trussworks/terraform-module-template/pull/8